proto/api.proto

```diff
@@ -938,13 +944,26 @@ message StepProject {
 }
 
 message GetRecentBlockHashRequest {
-
 }
 
 message GetRecentBlockHashResponse {
   string blockHash = 1;
 }
 
+message Block {
+  uint64 slot = 1;
+  string hash = 2;
+  int64 time = 3;
+  uint64 height = 4;
+}
+
+message GetBlockStreamRequest {
```

Comment:
I mentioned this in a follow comment here: https://github.com/bloXroute-Labs/solana-geyser-broadcast/pull/24 but I think we need commitment levels in these requests

Reply:
https://github.com/bloXroute-Labs/solana-geyser-broadcast/pull/24#discussion_r1037041444

Reply:
client side = Trader API or in client SDKs? Main problem is that transaction submission really favors more conservative commitment levels over faster ones, since there's the possibility that a fast commitment block doesn't end up getting accepted and the transaction gets rejected as a result. Do you know what the equivalent commitment level is that geyser is publishing now?

Reply:
if we do it in trader API we'll kinda be stuck doing a polling type thing, which I'm not sure is any better than polling `getLatestBlockhash` and `getBlock`

Reply:
returned recentBlockHash stream to timed, no changed in this repo